xbmc/GUIInfoManager.cpp

```diff
@@ -6048,36 +6048,36 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     strLabel = StringUtils::Format("%2.1f dB", CAEUtil::PercentToGain(g_application.GetVolume(false)));
     break;
   case PLAYER_SUBTITLE_DELAY:
-    strLabel = StringUtils::Format("%2.3f s", g_application.m_pPlayer->GetVideoSettings().m_SubtitleDelay);
+    strLabel = StringUtils::Format("%2.3f s", g_application.GetAppPlayer()->GetVideoSettings().m_SubtitleDelay);
     break;
   case PLAYER_AUDIO_DELAY:
-    strLabel = StringUtils::Format("%2.3f s", g_application.m_pPlayer->GetVideoSettings().m_AudioDelay);
+    strLabel = StringUtils::Format("%2.3f s", g_application.GetAppPlayer()->GetVideoSettings().m_AudioDelay);
     break;
   case PLAYER_CHAPTER:
-    if(g_application.m_pPlayer->IsPlaying())
-      strLabel = StringUtils::Format("%02d", g_application.m_pPlayer->GetChapter());
+    if(g_application.GetAppPlayer()->IsPlaying())
+      strLabel = StringUtils::Format("%02d", g_application.GetAppPlayer()->GetChapter());
     break;
   case PLAYER_CHAPTERCOUNT:
-    if(g_application.m_pPlayer->IsPlaying())
-      strLabel = StringUtils::Format("%02d", g_application.m_pPlayer->GetChapterCount());
+    if(g_application.GetAppPlayer()->IsPlaying())
+      strLabel = StringUtils::Format("%02d", g_application.GetAppPlayer()->GetChapterCount());
     break;
   case PLAYER_CHAPTERNAME:
-    if(g_application.m_pPlayer->IsPlaying())
-      g_application.m_pPlayer->GetChapterName(strLabel);
+    if(g_application.GetAppPlayer()->IsPlaying())
```

Comment:
with regard on performance we should think about a better solution. now that's 2 function calls extra and 2 times a shared ptr get constructed/destructed.